analyzer/codechecker_analyzer/analyzers/clangsa/statistics_collector.py

```diff
@@ -65,11 +65,25 @@ def build_stat_coll_cmd(action, config, source):
         cmd.extend(['-Xclang',
                     '-analyzer-checker=' + coll_check])
 
-    if action.target != "":
-        cmd.append("--target=" + action.target)
+    def has_flag(flag):
```

Comment:
This is defined also in `ctu_triple_arch.py`. Would it be possible to define only once?